index.js

```diff
@@ -16,15 +21,56 @@ async function run() {
     const logConfigurationOptions = core.getInput("log-configuration-options", { required: false });
     const dockerLabels = core.getInput('docker-labels', { required: false });
     const command = core.getInput('command', { required: false });
+    
+     //New inputs to fecth task definition 
+     const taskDefinitionArn = core.getInput('task-definition-arn', { required: false });
+     const taskDefinitionFamily = core.getInput('task-definition-family', { required: false });
+     const taskDefinitionRevision = Number(core.getInput('task-definition-revision', { required: false}));
 
-    // Parse the task definition
-    const taskDefPath = path.isAbsolute(taskDefinitionFile) ?
-      taskDefinitionFile :
-      path.join(process.env.GITHUB_WORKSPACE, taskDefinitionFile);
-    if (!fs.existsSync(taskDefPath)) {
-      throw new Error(`Task definition file does not exist: ${taskDefinitionFile}`);
+
+    if(taskDefinitionFile && taskDefinitionArn && taskDefinitionFamily){ 
```

Comment:
The current if statement requires all three fields to be provided(task definition file, arn and family/name) for the warning to be issued. Since, we want this if condition to execute even if only task definition file and task definition arn is provided (family name not provided), maybe we should change this be something like below:

```
(taskDefinitionFile && taskDefinitionArn) || (taskDefinitionFile && taskDefinitionFamily) {
...
} 
```